<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgCabecalhoBeneficio</class>
 <widget class="QWidget" name="wdgCabecalhoBeneficio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-------------------------------- Labels -----------------------------------------*/
#lbCdEmp {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 24px;
	font-weight: 900;
}

#lbDataInicio, #lbDataFim, #lbSituacao {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 16px;
}

#lbInfoDataInicio, #lbInfoDataFim, 
#lbInfoSituacao, #lbInfoNb {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 14px;
	color: white;
	text-decoration: underline;
}

/* --------------------------------- Frames --------------------------------- */
#frMain {
	/*background-color: white;*/
	background-color: qlineargradient(spread:pad, x1:0.535, y1:0.761364, x2:0.296122, y2:0.415, stop:0 rgba(58, 171, 140, 255), stop:1 rgba(9, 212, 156, 255));
	border-radius: 8px;
/*	border: 1px solid grey;*/
}

#frIcone {
	background: url(:/beneficio/beneficio.png);
	background-repeat: no-repeat;
	background-position: center;
}


/* --------------------------------- Push Button --------------------------------- */
#pbConfirmar {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 14px;
	font-weight: 600;
	color: white;

	border-radius: 4px;
	border: 2px solid white;
	background-color: transparent;
}

#pbConfirmar:hover {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 14px;
	font-weight: 600;
	color: white;

	border-radius: 4px;
	border: 2px solid white;
	background-color: rgba(85, 87, 83, 70);
}

#pbCancelar {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 14px;
	font-weight: 600;
	color: rgb(60, 60, 60);

	border-radius: 4px;
	border: 2px solid rgb(60, 60, 60);
	background-color: transparent;
}

#pbCancelar:hover {
	font-family: &quot;TeX Gyre Adventor&quot;;
	font-size: 14px;
	font-weight: 600;
	color: rgb(60, 60, 60);

	border-radius: 4px;
	border: 2px solid rgb(60, 60, 60);
	background-color: rgba(255, 255, 255, 120);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>680</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frIcone">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>16</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="cbxNomeBeneficio"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="lbInfoNb">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Número do benefício:</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="leNb">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>171</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="lbInfoDataInicio">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>91</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Data início:</string>
               </property>
              </widget>
              <widget class="QDateEdit" name="dtDataInicio">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>110</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="lbInfoDataFim">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>10</y>
                 <width>64</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Data fim:</string>
               </property>
              </widget>
              <widget class="QDateEdit" name="dtDataFim">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>30</y>
                 <width>110</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="lbInfoSituacao">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>81</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Situação:</string>
               </property>
              </widget>
              <widget class="QComboBox" name="cbxSituacao">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>131</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>400</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbConfirmar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Confirmar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCancelar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/itemResumo.qrc"/>
 </resources>
 <connections/>
</ui>
